ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -83,12 +74,26 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-arm64-p2-f14
+  cron: 9 14 5,20 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
+      ADDITIONAL_WORKER_VM_TYPE: m6i.xlarge
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      OCP_ARCH: arm64
```

Comment:
Per the error message in https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/36173/rehearse-36173-periodic-ci-openshift-openshift-tests-private-release-4.13-multi-nightly-4.13-upgrade-from-stable-4.12-aws-ipi-arm64-p2-f14/1625679162758926336:
```
level=error msg=Error: creating EC2 Instance: InvalidParameterValue: The architecture 'x86_64' of the specified instance type does not match the architecture 'arm64' of the specified AMI. Specify an instance type and an AMI that have matching architectures, and try again. You can use 'describe-instance-types' or 'describe-images' to discover the architecture of the instance type or AMI.
```

`COMPUTE_NODE_TYPE` and `CONTROL_PLANE_INSTANCE_TYPE` need to be overwritten, because by default, `cucushift-installer-rehearse-aws-ipi-provision` set them to x86 instance type.

Reply:
ah, thank you Jianlin for the debug!!